libs/utils/env.py

```diff
@@ -603,8 +604,13 @@ def _init_platform_smp(self):
             # Try loading frequencies using the cpufreq module
             for cluster_id in self.platform['clusters']:
                 core_id = self.platform['clusters'][cluster_id][0]
-                self.platform['freqs'][cluster_id] = \
-                    self.target.cpufreq.list_frequencies(core_id)
+                try:
+                    self.platform['freqs'][cluster_id] = \
+                        self.target.cpufreq.list_frequencies(core_id)
+                except TargetError:
```

Comment:
I'm not comfortable with a blanket `except TargetError`. We still want an error here if an ARM target is misconfigured.

How about either just checking `target.abi` or checking (in devlib?) if `scaling_driver` is `intel_pstate`?